ngraph/core/shape_inference/include/depth_to_space_shape_inference.hpp

```diff
@@ -0,0 +1,57 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <cstdint>
+#include <openvino/core/validation_util.hpp>
+#include <openvino/op/depth_to_space.hpp>
+#include <openvino/opsets/opset1.hpp>
+
+#include "utils.hpp"
+namespace ov {
+namespace op {
+namespace v0 {
+
+template <class T>
+void shape_infer(const ov::op::v0::DepthToSpace* op,
+                 const std::vector<T>& input_shapes,
+                 std::vector<T>& output_shapes) {
+    using ValType = typename std::iterator_traits<typename T::iterator>::value_type::value_type;
+
+    NODE_VALIDATION_CHECK(op, input_shapes.size() == 1 && output_shapes.size() == 1);
+
+    const auto& data_shape = input_shapes[0];
+    const ov::Rank data_rank = data_shape.rank();
+    const auto block_size = op->get_block_size();
```

Comment:
Please make shape_infer function a private friend and correct this line
```suggestion
    const auto& block_size = op->m_blocksize;
```

Reply:
change to  const aut&  block_size = op->get_block_size();